styles/core/forms.less

```diff
@@ -13,25 +13,21 @@
 * not, you can obtain one from Tidepool Project at tidepool.org.
 */
 
-// Form controls
-// ====================================
+@import 'variables.less';
 
-@form-control-border-size:   2px;
-@form-control-height:        (30px + 2*@form-control-border-size);
-@form-control-border-color:  @purple-light;
 
 .form-control-border {
```

Comment:
The forms.less file is still depended on by the dropdown.less in a traditional LESS way, so I didn't want to go changing all the classnames unnecessarily. The forms are also never directly required by a JS file, so the advantage of being able to reference the class name directly as a simple object property via the dot syntax vs using the array index string format isn't really there. I can add something in the doc concerning that.